openrtb-json/src/test/scala/com/powerspace/openrtb/json/OpenRtbSerdeTest.scala

```diff
@@ -0,0 +1,49 @@
+package com.powerspace.openrtb.json
+
+import java.net.URL
+import com.google.openrtb.{BidRequest, BidResponse}
+import io.circe.parser._
+import io.circe.syntax._
+import org.scalatest.{FunSuite, GivenWhenThen}
+
+class OpenRtbSerdeTest extends FunSuite with GivenWhenThen {
+
+  import OpenRtbSerdeModule._
+
+  test("OpenRTB-like (Elastic Ads) bid response deserialization") {
```

Comment:
Please provide a test for NoBid cases (both BS and plain OPENRTB)

Reply:
oki-doki